runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerDoseOverlay.java

```diff
@@ -114,11 +112,19 @@ public Dimension render(Graphics2D graphics)
 
 		if (config.showPrayerStatistics() && bounds.contains(mousePosition.getX(), mousePosition.getY()))
 		{
-			final String tooltip = "Time Remaining: " + getEstimatedTimeRemaining() +
-				"</br>" +
-				"Prayer Bonus: " + prayerBonus;
+			final StringBuilder sb = new StringBuilder();
 
-			tooltipManager.add(new Tooltip(tooltip));
+			if (config.replaceOrbText())
+			{
+				sb.append("Prayer points remaining: " + client.getBoostedSkillLevel(Skill.PRAYER));
```

Comment:
```suggestion
				sb.append("Prayer points remaining: ".append(client.getBoostedSkillLevel(Skill.PRAYER));
```

Reply:
Don't you mean `sb.append(str1).append(str2)`? Not `sb.append(str1.append(str2))`?